src/main/java/core/basesyntax/service/strategy/impl/TransactionProcessorImpl.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.service.strategy.impl;
```

Comment:
this is not realy part of the strategy, this class just uses it as a component
```suggestion
package core.basesyntax.service.impl;
```